### ISSUE TITLE
Mod Suggestion: Floodlights

### ISSUE THREAD

**Issue by TonyTash** [Electrical-Age/ElectricalAge#497]
Hello, I think it would be useful if the mod had a floodlight integration, similar to the floodlight in immersive engineering and kerosene lamp in pneumaticcraft. 

It would probably light up a range of 20ish blocks, depending on the bulb used in the lamp. It would probably also light up the area in a single direction, rather then all around it.

It could also probably use Halogen, HPS or MH/MV bulbs (Refer to https://github.com/Electrical-Age/ElectricalAge/issues/407 )

(these could just be long range bulbs that you can put into any light, but there could also be another type of light)

As for 3D models, I was thinking about something like this for a wall mounted variant: 
http://bit.ly/2a7e3Nv
http://thd.co/2a2HpfX

(OPTIONAL) Or, maybe on a stand such as this (maybe even the ability to mount 2 on a stand, to light up a larger area): 
http://bit.ly/2akRes1

**Comment by bowserjr2424** [comment]
Isn't that first one a wall pack?

**Comment by TonyTash** [comment]
It was (changed it now), but it could still be used as a flood light. I was thinking about something more like the second one, or like the LED floodlights like http://bit.ly/2a7e3Nv

**Comment by Baughn** [comment]
If we're adding longer-range lights, it would be nice if the lights in general were at least as good as torches. 50V electric lights shouldn't be _worse_ than what they're supposed to be replacing, and 200V should be better than that again.

**Comment by TonyTash** [comment]
Well, if they are fixed, I think that it would be a useful addition to have some sort of long-range light

**Comment by iancbogue** [comment]
Yeah. Seeing some more powerful bulbs would be nice to see because I have a building that I'm making that has 16-block high walls, so the lighting looks spotty. Maybe 20, 25, and 30 block ranges, anyone?

**Comment by mattmtgaming005** [comment]
``

**Comment by AMIDIBOSS** [comment]
Okay, I've the model. It's Floorstanding, but i'll make Wallmount if you wish (It's basicly to remove the stand)

**Comment by iancbogue** [comment]
Wallmounted would be amazing! Are you basing your model off any of the ones linked by TonyTash?

**Comment by AMIDIBOSS** [comment]
Well, Every wallmount floodlight looks pretty much the same so yeah.
![spotlight](https://cloud.githubusercontent.com/assets/10089795/21194138/21d592e2-c22f-11e6-96bb-85a7460d2e3a.png)
It'll propably look bether in-game

**Comment by iancbogue** [comment]
OK. Looks nice. Can't wait!
-- 
Ian
